### ISSUE TITLE
uniformal page on the build process needs update

### ISSUE THREAD

**Issue by kohlhase** [UniFormal/MMT#267]
https://uniformal.github.io/doc/applications/building.html
this still speaks of `mmt build` where it is now `mmt make`, and does not mention the 

Similarly for https://uniformal.github.io/doc/applications/stex.html that needs full rework. (and integration into the documentation for informal content.

**Comment by Jazzpirate** [comment]
No, this seems perfectly accurate to me. The shell command(!) for building a build target is in fact "build".
I don't know what exactly "mmt make" does, but it must be a different command.
I can't comment on stex and latexml ;)
